en/installation-guide/database-initialization.rst

```diff
@@ -12,7 +12,7 @@ You can use :ref:`pgAdmin` to access and manage the database server.
 Process
 -------
 
-In pgAdminIII
+In pgAdminIII or pgAdmin4
```

Comment:
I think we could switch everything to pgAdmin4. We better have one clear recommendation (and those who want to stick to pgAdmin3 can do that anyway)

Reply:
ok, should be done - just leaving the version away.